pkg/backup/backup.go

```diff
@@ -71,6 +73,7 @@ func Run(client controller.Client, out io.Writer, progress ProgressBar) error {
 				"MONGO_PWD": mongodbRelease.Env["MONGO_PWD"],
 			},
 			DisableLog: true,
+			Partition:  "background",
```

Comment:
Is it worth having constants for these (e.g. `ct.PartitionTypeBackground` etc.)?